src/ts/component/block/featured.tsx

```diff
@@ -677,6 +682,34 @@ const BlockFeatured = observer(class BlockFeatured extends React.Component<Props
 		});
 	};
 
+	onDate (e: React.MouseEvent, relationKey: string) {
+		const { rootId } = this.props;
+		const storeId = this.getStoreId();
+		const object = detailStore.get(rootId, storeId, [ relationKey ]);
+		const relation = dbStore.getRelationByKey(relationKey);
+		const value = object[relationKey] || UtilDate.now();
+		const element = $(e.currentTarget).find('.cellContent');
```

Comment:
Remove e.currentTarget, change to const elementId = Relation.cellId(PREFIX + block.id, relationKey, object.id);